src/main/java/jenkins/plugins/office365connector/FactsBuilder.java

```diff
@@ -104,7 +110,8 @@ public void addRemarks(List<Cause> causes) {
         facts.add(new Fact(NAME_REMARKS, causesStr.toString()));
     }
 
-    public void addCulprits(Result result, Set<User> authors) {
+    public void addCulprits(Set<User> authors) {
```

Comment:
prefers previous but can't give strong reason for that

Reply:
I can add culprits is about the users who made it go red, result is only there for the fact that there are no culprits.

Reply:
https://github.com/jenkinsci/jenkins/blob/e952bbaff332ff6a28ccdca43e285fb58f6a6572/core/src/main/java/jenkins/scm/RunWithSCM.java#L90-L139 I see no reason who check result here.

Reply:
https://github.com/jenkinsci/jenkins/blob/08def67a18eee51de9f3f99bc2a792fee1c160e0/core/src/main/java/hudson/model/AbstractBuild.java#L334-L382